include/swift/Basic/LangOptions.h

```diff
@@ -357,11 +357,15 @@ namespace swift {
     bool EnableTargetOSChecking = true;
 
     /// Whether to attempt to recover from missing cross-references and other
-    /// errors when deserializing from a Swift module.
+    /// errors when deserializing from a binary swiftmodule file.
     ///
-    /// This is a staging flag; eventually it will be removed.
+    /// This flag should only be used in testing.
```

Comment:
Seems this isn't actually true since we need it enabled in order to have `EnableDeserializationSafety` work. Plus at the very least LLDB would like want this regardless.

Reply:
Oh right, thanks for pointing it out, it should say `This flag should only be disabled in testing.`. We always want this enabled unless we want to cause a crash.